php/php.editor/src/org/netbeans/modules/php/editor/codegen/CGSInfo.java

```diff
@@ -251,9 +255,9 @@ private void initProperties(ResultIterator resultIterator) throws ParseException
         PHPParseResult info = (PHPParseResult) resultIterator.getParserResult();
         if (info != null) {
             int caretOffset = textComp.getCaretPosition();
-            TypeDeclaration typeDecl = findEnclosingClassOrTrait(info, caretOffset);
+            ASTNode typeDecl = findEnclosingType(info, caretOffset);
```

Comment:
Why this change exactly? Because of Enums? If so, Enum is not a TypeDeclaration?

Reply:
Because of anonymous classes(`ClassInstanceCreation`).

Reply:
I see now, thanks.